### PR TITLE
1660 - fixing the colors on the alerts

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

File: web-client/src/styles/variables.scss [hunk 1]
```diff
@@ -19,6 +19,9 @@ $color-primary-alt: #61e5ff;
 $color-primary-alt-lightest: #f1f9fc;
 $color-white: #fff;
 $color-green: #2e8540;
-
+$color-success: #e7f4e4;
+$color-red: #d83933;
```
Comment (rachaelparris) [inline web-client/src/styles/variables.scss]: There's a color variable for this red: `$theme-color-secondary`.

File: web-client/src/styles/variables.scss [hunk 2]
```diff
@@ -19,6 +19,9 @@ $color-primary-alt: #61e5ff;
...
+$color-error: #f8dfe2;
```
Comment (rachaelparris) [inline web-client/src/styles/variables.scss]: This one is `$theme-color-secondary-lighter`.

File: web-client/src/styles/variables.scss [hunk 3]
```diff
@@ -19,6 +19,9 @@ $color-primary-alt: #61e5ff;
...
+$color-info-darker: #28a0cb;
```
Comment (rachaelparris) [inline web-client/src/styles/variables.scss]: This one is `$theme-color-accent-cool-dark`.

**Review by rachaelparris — CHANGES_REQUESTED** [review]
A few of the colors you added are in the USWDS color variables here: https://v2.designsystem.digital.gov/design-tokens/color/theme-tokens/

**Review by kkoskelin — APPROVED** [review]

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**